<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sigconsmed</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Qt.png</normaloff>Qt.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#verticalSpacer {
	height: 50px;
}
#panel, #MainWindow{
	background-color:#263238;
}
#panel{
	margin: 0px;
}
#pb_agenda,#pb_turnos, #pb_medicos, #pb_pacientes, #pb_secretarios, #pd_logOut{
	font-size: 12px;
	background-color:#263238;
	border:none;
	color: white;
	margin: 0px 0px;
	text-align: center;
	height:80px;
    width: 130px;
}
#pb_agenda {
	background-color: #00796b;
}

#pb_medicos:hover,#pb_turnos:hover,
#pb_pacientes:hover, #pb_secretarios:hover{
	background-color: #00796b;
}

#pd_logOut:hover,
#pd_logOut:focus {
	background-color:#d32f2f;
}
#L_userPhoto{
	background: url(icon/user.png);
  	border-radius: 40px;
}
#L_userName{
	font-size: 12px;
	color: white;
}
#logo{
	min-width:150px;
	min-height:50px;
	max-width:150px;
	max-height:50px;
	margin-top:5px
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1" alignment="Qt::AlignRight|Qt::AlignTop">
     <widget class="QWidget" name="w_user" native="true">
      <layout class="QGridLayout" name="gridLayout_2"/>
     </widget>
    </item>
    <item row="1" column="0" rowspan="3">
     <widget class="QWidget" name="panel" native="true">
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="logo">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>Sigconsmed.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>30</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pb_medicos">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Doctores</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>doctor.png</normaloff>doctor.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_pacientes">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Pacientes</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>pacientes.png</normaloff>pacientes.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_turnos">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string> Consultas</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>organize.png</normaloff>organize.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_agenda">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Incisos</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Incisos.png</normaloff>Incisos.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pd_logOut">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>exit.png</normaloff>exit.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Sigconsmed beta v0.1  </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string> Diseñado y Desarrollado por Legner José Morales Manso  </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#ffffff;&quot;&gt;Sistema de Gestión de Consultas Médicas&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QMdiArea" name="mdiArea">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <tabstops>
  <tabstop>pb_agenda</tabstop>
  <tabstop>pb_turnos</tabstop>
  <tabstop>pb_pacientes</tabstop>
  <tabstop>pb_medicos</tabstop>
  <tabstop>pd_logOut</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
